2022.12.4

balloob opened the pull request:
- Bump bluetooth-auto-recovery to 1.0.0 ([@bdraco] - [#83800]) ([bluetooth docs])
- Update frontend to 20221212.0 ([@bramkragten] - [#83842]) ([frontend docs])
- Bump version of ZHA quirks to 0.0.89 ([@dmulcahey] - [#83848]) ([zha docs])
- Bump pychromecast to 13.0.3 ([@emontnemery] - [#83861]) ([cast docs])
- Move template check into fritzbox entry setup ([@mib1185] - [#83863]) ([fritzbox docs])
- Use the async_migrate_paypal_agreement function to get the migration URL ([@ludeeus] - [#83469]) ([cloud docs])
- Fix issue on Overkiz Domestic Hot water heater entities with away mode ([@nyroDev] - [#83684]) ([overkiz docs])
- Bump pyoverkiz to 1.7.2 ([@nyroDev] - [#83866]) ([overkiz docs])
- Fix bleak-retry-connector get_device hang with HAOS 9.4 and only proxies ([@bdraco] - [#83879]) ([bluetooth docs])
- Fix HomeKit media players when entity has duplicate sources ([@bdraco] - [#83890]) ([homekit docs])

[#83469]: https://github.com/home-assistant/core/pull/83469
[#83482]: https://github.com/home-assistant/core/pull/83482
[#83592]: https://github.com/home-assistant/core/pull/83592
[#83684]: https://github.com/home-assistant/core/pull/83684
[#83778]: https://github.com/home-assistant/core/pull/83778
[#83797]: https://github.com/home-assistant/core/pull/83797
[#83800]: https://github.com/home-assistant/core/pull/83800
[#83842]: https://github.com/home-assistant/core/pull/83842
[#83848]: https://github.com/home-assistant/core/pull/83848
[#83861]: https://github.com/home-assistant/core/pull/83861
[#83863]: https://github.com/home-assistant/core/pull/83863
[#83866]: https://github.com/home-assistant/core/pull/83866
[#83879]: https://github.com/home-assistant/core/pull/83879
[#83890]: https://github.com/home-assistant/core/pull/83890
[@balloob]: https://github.com/balloob
[@bdraco]: https://github.com/bdraco
[@bramkragten]: https://github.com/bramkragten
[@dmulcahey]: https://github.com/dmulcahey
[@emontnemery]: https://github.com/emontnemery
[@frenck]: https://github.com/frenck
[@ludeeus]: https://github.com/ludeeus
[@mib1185]: https://github.com/mib1185
[@nyroDev]: https://github.com/nyroDev
[abode docs]: https://www.home-assistant.io/integrations/abode/
[accuweather docs]: https://www.home-assistant.io/integrations/accuweather/
[bluetooth docs]: https://www.home-assistant.io/integrations/bluetooth/
[cast docs]: https://www.home-assistant.io/integrations/cast/
[cloud docs]: https://www.home-assistant.io/integrations/cloud/
[fritzbox docs]: https://www.home-assistant.io/integrations/fritzbox/
[frontend docs]: https://www.home-assistant.io/integrations/frontend/
[homekit docs]: https://www.home-assistant.io/integrations/homekit/
[overkiz docs]: https://www.home-assistant.io/integrations/overkiz/
[zha docs]: https://www.home-assistant.io/integrations/zha/

ludeeus committed:
Use the async_migrate_paypal_agreement function to get the migration URL (#83469)

* Use the async_migrate_paypal_agreement function to get the migration URL

* Update URL

* Handle timeout error

balloob committed:
SleepIQ to not retry a re-auth right away

bdraco committed:
Bump bluetooth-auto-recovery to 1.0.0 (#83800)

fixes https://github.com/home-assistant/core/issues/83673

dmulcahey committed:
Bump version of ZHA quirks to 0.0.89 (#83848)

bramkragten committed:
Update frontend to 20221212.0 (#83842)

emontnemery committed:
Bump pychromecast to 13.0.3 (#83861)

mib1185 committed:
Move template check into fritzbox entry setup (#83863)

* move template check into entry setup

* use else in try-except block

balloob committed:
Bumped version to 2022.12.4

nyroDev committed:
Fix issue on Overkiz Domestic Hot water heater entities with away mode (#83684)

* Don't call set boostMode Duration with 0 to disable boost mode

* use states.get() instead of states[]

* Revert select states

nyroDev committed:
Bump pyoverkiz to 1.7.2 (#83866)

Bump pyoverkiz

bdraco commented:
one more that would be nice to get in https://github.com/home-assistant/core/pull/83879

bdraco committed:
Fix HomeKit media players when entity has duplicate sources (#83890)

fixes #83852 fixes #83698

bdraco committed:
Fix bleak-retry-connector get_device hang with HAOS 9.4 and only proxies (#83879)

The bleak connection setup to dbus hangs forever instead of errors if
there are no local bluetooth adapters and we are using remote proxies.

The library now times out when this happens instead of hanging
forever.

changelog: https://github.com/Bluetooth-Devices/bleak-retry-connector/compare/v2.10.1...v2.10.2

balloob commented:
I forgot I didn't merge this one yet, will update it with recently merged tings.
